index.bs

```diff
@@ -516,6 +516,10 @@ The sample code for generating and registering a new key follows:
         {
           type: "public-key",
           alg: -257 // Value registered by this specification for "RS256"
+        },
+        {
+          type: "public-key",
+          alg: -8 // "EdDSA" as registered in the IANA COSE Algorithms registry
```

Comment:
I think EdDSA should be first as this is processed as most preferred to least preferred.